CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 Changes
 =======
 
+Version 0.3.2 (2018-09-25)
+--------------------------
+- Modifies OS related commands for CephFS compatibility.
```

Comment:
Authors ORCID's instead of emails missing. Regarding the change log, shall we add space between the title and the list of changes like we do in other repos https://github.com/reanahub/reana-client/commit/d887d0f354bb76d64516acf6755a0dfcac52511b?